README.md

```diff
@@ -79,8 +79,10 @@ published. This follows the format `<name>=<value>`.
 
 - `--no-secure` if you want your logs to be sent to the Insight Platform un-encrypted (no TSL/SSL).  
 
-- `--debug` if you want the container to print/log its debug information for trouble-shooting
```

Comment:
considering we've already given this option to support and some customers, it might make a few people's lives (including ours, as a result) a bit easier if we continue to support this as an alias for `loggerlevel=debug`.

Reply:
Nah, feature creep, I'll write up a page about the breaking changes.
These images are versioned and use semver and we're bumping major

Reply:
let's not make breaking changes if we don't have to. add deprecation notices to the logging output if you want, but this is just needlessly going to make our lives more difficult.

Reply:
this is a necessary breaking change with what I'm bringing in to specify log levels, it's just confusing to allow both `debug` and `log-level`